### PR TITLE
Added read me

### PR DESCRIPTION
**Pull request by f7deleon**
Added read me

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Suspended Location
+![Check and compile](https://github.com/xmartlabs/SuspendedLocation/workflows/Check%20and%20compile/badge.svg?branch=master)
+[![Release](https://jitpack.io/v/xmartlabs/AndroidSwissKnife.svg)](https://jitpack.io/#xmartlabs/SuspendedLocation)
+
+A library that wraps the google [places](https://developers.google.com/places/android-sdk/overview) and [geolocation](https://developers.google.com/android/reference/com/google/android/gms/location/package-summary) Apis through coroutines
```
Comment (agussante16) [inline README.md:5]:
```suggestion
A library that wraps the google [places](https://developers.google.com/places/android-sdk/overview) and [geolocation](https://developers.google.com/android/reference/com/google/android/gms/location/package-summary) APIs through coroutines
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+It's divided into a series of modules with different proposes:
```
Comment (agussante16) [inline README.md:7]:
```suggestion
It's divided into a series of modules with different purposes:
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+- [Core](/core): Contains functions that wraps [geolocation](https://developers.google.com/android/reference/com/google/android/gms/location/package-summary) api.
```
Comment (agussante16) [inline README.md:8]:
```suggestion
- [Core](/core): Contains functions that wrap [geolocation](https://developers.google.com/android/reference/com/google/android/gms/location/package-summary) API.
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+- [Places](/places): Contains functions that wraps [places](https://developers.google.com/places/android-sdk/overview) api.
```
Comment (agussante16) [inline README.md:9]:
```suggestion
- [Places](/places): Contains functions that wrap [places](https://developers.google.com/places/android-sdk/overview) API.
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+## Setup
+Add library to project dependencies with JitPack.
+
+```groovy
+repositories {
+    maven { url "https://jitpack.io" }
+}
+
+dependencies {
+    implementation "com.github.xmartlabs.SuspendedLocation:core:${master-latest-hash-commit}"
```
Comment (matir91) [inline README.md:20]: I would not use the lastest hashes, I'd use specific versions

**Pull request merged**